ansible/configs/ocp4-workshop/lifecycle.yml

```diff
@@ -91,7 +92,7 @@
           template:
             dest: "{{ output_dir }}/status.txt"
             src: files/status.j2
-       
```

Comment:
The filters should contain instance-state-name: stopped - see original code in fork.

Reply:
This was added to your fork in the same commit that added the `instance-state-name: running` on line 71. (https://github.com/jimrigsbee/agnosticd/commit/c4f30e10453dc5a8aaf37f80da7dee72e5dc191a#diff-fc4f0c6c43bc16706a2a449e4f5a9176). Since this was one of the very few files that Razique and I made a couple of manual changes, this looks to me like something that happened when we were editing it rather than a bigger issue.